src/renderer/components/common/QrCode/common/types.ts

```diff
@@ -50,7 +55,25 @@ export type AddressInfo = {
   genesisHash: Uint8Array;
 };
 
+export type DdAddressInfo = {
+  publicKey: MultiSigner;
+  derivationPath: string;
+  encryption: CryptoType;
+};
+
 export type ErrorObject = {
   code: QrError;
   message: string;
 };
+
+export type DdAddressInfoDecoded = {
+  publicKey: { MultiSigner: Exclude<CryptoTypeString, CryptoTypeString.ETHEREUM>; public: string };
```

Comment:
maybe create type for this? Like `MultiSigner` that we already have, or maybe use it with type of public `Uint8Array | string`